containers/Connector/Connector.tsx

```diff
@@ -26,8 +31,9 @@ const useConnector = () => {
 	const [onboard, setOnboard] = useState<ReturnType<typeof initOnboard> | null>(null);
 	const [notify, setNotify] = useState<ReturnType<typeof initNotify> | null>(null);
 	const [isAppReady, setAppReady] = useRecoilState(appReadyState);
-	const setWalletAddress = useSetRecoilState(walletAddressState);
+	const [walletAddress, setWalletAddress] = useRecoilState(walletAddressState);
 	const [walletWatched, setWalletWatched] = useRecoilState(walletWatchedState);
+	const [isEOAWallet, setIsEOAWallet] = useRecoilState(isEOAWalletState);
```

Comment:
Since `isEOAWallet` is not used here, you may want to use `useSetRecoilState` instead.
`const setIsEOAWallet = useSetRecoilState(isEOAWalletState)`

when `useRecoilState` is used, it would subscribe to any changes of `isEOAWallet`, so in those cases, you just want `useSetRecoilState`.

Reply:
good point